typescript-playwright-sample/README.md

```diff
@@ -0,0 +1,73 @@
+# typescript-playwright-sample
+
+This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser, including how to suppress pre-existing or third-party failures. 
+
+This sample uses [Playwright](https://playwright.dev) for browser automation and uses the corresponding `@axe-core/playwright` library to integrate Axe and Playwright. But you don't have to use Playwright to use Axe! If you prefer a different browser automation tool, you can still follow the same concepts from this sample by using the integration library appropriate for your framework:
+
+* For **Selenium**, see our [typescript-selenium-webdriverjs-sample](../typescript-selenium-webdriverjs-sample/README.md)
+* For **Puppeteer**, use [@axe-core/puppeteer](https://www.npmjs.com/package/@axe-core/puppeteer)
+* For **Cypress**, use [cypress-axe](https://www.npmjs.com/package/cypress-axe)
+* For **WebdriverIO**, see our [typescript-selenium-webdriverjs-sample](../typescript-selenium-webdriverjs-sample/README.md) and use [@axe-core/webdriverio](https://www.npmjs.com/package/@axe-core/webdriverio)
+* For **Protractor**, see our [typescript-selenium-webdriverjs-sample](../typescript-selenium-webdriverjs-sample/README.md); keep using `@axe-core/webdriverjs` like that sample shows, but instead of creating your own Webdriver object, pass [`browser.webdriver`](https://www.protractortest.org/#/api?view=ProtractorBrowser) to `@axe-core/webdriverjs`.
+
+## Getting Started
+
+The individual files in the sample contain comments that explain the important parts of each file in context.
+
+Some good places to start reading are:
+
+* [tests/index.spec.ts](./tests/index.spec.ts): Playwright test file that opens [src/index.html](./src/index.html) in a browser with Playwright and runs accessibility scans against it with `@axe-core/playwright`
+* [azure-pipelines.yml](./azure-pipelines.yml): Azure Pipelines config file that sets up our Continuous Integration and Pull Request builds
+* [playwright.config.js](./playwright.config.js): Playwright configuration file that enables test result reporting in Azure Pipelines (using the `junit` reporter)
+
+## Tools and libraries used
+
+* [TypeScript](https://www.typescriptlang.org/) to author our test code
+* [Playwright](https://playwright.dev/) as our test framework and browser automation library
+* [@axe-core/playwright](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/playwright) to run an axe accessibility scan on the page from the Playwright browser
+* [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to run the tests in a CI build with every Pull Request
+* [axe-sarif-converter](https://github.com/microsoft/axe-sarif-converter) to convert axe results to SARIF format
+* [Sarif Viewer Build Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab) to visualize the results in Azure Pipelines
```

Comment:
The description in this link points to a newer version of the extension - https://marketplace.visualstudio.com/items?itemName=sariftools.scans should we link there directly?

Reply:
Yes, but I wanted to leave updating to that for a separate PR since it affects the other samples too

Reply:
Sounds good!